### DISCUSSION TITLE
Inconsistent git installation and website results

### DISCUSSION THREAD

**Discussion by blackbin7** [visioncortex/vtracer#94] [Q&A]
Hi, I've tested the same graph in two separate methods using pip local install and https://www.visioncortex.org/vtracer/, but the results will be different (when binarising thin lines to vectors). The results from the website are significantly better (more detail) than the local, and I would like to ask if there is any difference between the two methods? 
<img width="1508" alt="image" src="https://github.com/user-attachments/assets/eb16ff55-3975-4fe6-9276-41946ca96918">

**Comment by tyt2y3** [comment]
technically they are the same code. but the cli version's (hence python binding) parameters are slightly different.

so I guess you need to try different parameters.

and to me it seems like a resolution problem of the input images
